### PR TITLE
[Dashboard] Actor Table UI Optimize

### PR DESCRIPTION
**Pull request by mxz96102**
## Why are these changes needed?

Refresh on actor page: #13883
Make num_restarts a separate field in the actors page: #13888
Remove the TaskFuncDesc and replace it with a human-readable name: #13893
Collapse actor id with copy button (because actor become longer and longer, display all is not that nessasery, instead we provide a copy button for user to copy it)

## Checks

- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: dashboard/client/package.json [hunk 1]
```diff
@@ -126,5 +127,5 @@
       "last 1 safari version"
     ]
   },
-  "proxy": "http://localhost:8265"
+  "proxy": "http://100.82.253.32:8090"
```
Comment (edoakes) [inline dashboard/client/package.json:130]: intentional?

**Comment by edoakes** [comment]
@simon-mo could you review this please?

File: dashboard/client/package.json (hunk 1 above)
Comment (mxz96102) [reply dashboard/client/package.json]: Sorry, that's a mistake commit. delete now

**Comment by simon-mo** [comment]
```
./src/components/CopyableCollapse.tsx
Cannot find module: 'copy-to-clipboard'. Make sure this package is installed.
```

Also we recently pinned prettier and eslint version, can you resolve the merge conflict? (at least we should keep pinning them to avoid breakage).

File: dashboard/client/src/components/CopyableCollapse.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import { Tooltip } from "@material-ui/core";
+import { CheckOutlined, FileCopyOutlined } from "@material-ui/icons";
+import copy from "copy-to-clipboard";
```
Comment (simon-mo) [inline dashboard/client/src/components/CopyableCollapse.tsx:3]: add to pacakge.json

File: dashboard/client/src/pages/actor/index.tsx [hunk 3]
```diff
@@ -1,4 +1,5 @@
-import { makeStyles } from "@material-ui/core";
+import { Grid, makeStyles, Switch } from "@material-ui/core";
+import dayjs from "dayjs";
```
Comment (simon-mo) [inline dashboard/client/src/pages/actor/index.tsx:2]: add to pacakge.json or do we have to use it? js date is covers majority of the use cases https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date

**Review by simon-mo — APPROVED** [review]

File: dashboard/client/src/pages/actor/index.tsx (hunk 3 above)
Comment (mxz96102) [reply dashboard/client/src/pages/actor/index.tsx:2]: Dayjs is a light-weight date library that can handle more types of time string input and more operates on time, it was add to package.json the first time new dashboard merged

**Comment by mxz96102** [comment]
> @mxz96102 This PR was reverted, can you re-open a PR and update the package-lock.json?

#16071 - Update package-lock.json and revert back the feature

**Pull request merged**